UI/Selenium/appsettings.json

```diff
@@ -48,7 +48,18 @@
       "DefaultElementWait": "20",
       "OneMinuteElementWait": "60",
       "ReportLocation": "/TestResults/Reports",
-      "ImageLocation": "/TestResults/Images"
+      "ImageLocation": "/TestResults/Images",
+      "RunOnSauceLabs": true,
+      "SauceLabsConfiguration": {
+        "SauceUsername": "VideoHearings",
+        "SauceUrl": "@ondemand.eu-central-1.saucelabs.com:80/wd/hub",
+        "BrowserName": "safari",
```

Comment:
can we have chrome browser included in iPhone as well ?

Reply:
this is one of the many configurations. We can change these settings to our needs